docset/winserver2012r2-ps/netsecurity/New-NetFirewallRule.md

```diff
@@ -305,17 +305,14 @@ Specifies that matching firewall rules of the indicated edge traversal policy ar
 
 This parameter specifies how this firewall rule will handle edge traversal cases.
 Valid only when the **Direction** parameter is set to Inbound. 
-The acceptable values for this parameter are: Block, Allow, DeferToUser, or DeferToApp. 
-
+The acceptable values for this parameter are Block and Allow.
```

Comment:
I was able to use "DeferToApp" and "DeferToUser" in WinServ2012R2 (the article you are editing belongs to that version), can you kindly detail what is the reason to remove those values from acceptable list? thanks